lnrpc/rpc.proto

```diff
@@ -1515,6 +1515,21 @@ message Peer {
 
     /// Features advertised by the remote peer in their init message.
     map<uint32, Feature> features = 11 [json_name = "features"];
+
+    /*
+    Latest error is the last error we received from the peer. This error is
+    tracked across peer connections, but is not currently persisted across lnd
+    restarts.
+    */
+    TimestampError latest_error = 12;
```

Comment:
I'd probably call this `TimestampedError`. Somehow `TimestampError` to me reads like there as an error with the timestamp :)

Reply:
I agree with the naming suggestion. I also think it would make sense to make this field `repeated` and send the whole list of errors if we go through all the trouble of caching them.